src/services/transcript-parser.js

```diff
@@ -693,22 +677,24 @@ function cleanTimedText(lines) {
 function groupTimedTextLines(lines) {
   let groups = [];
   let i;
-  for (i = 0; i < lines.length;) {
+  for (i = 0; i < lines.length; i++) {
     const line = lines[i];
     let t = {};
-    if (line.includes('-->') || (/^NOTE/).test(line.toUpperCase())) {
-      const isNote = (/^NOTE/).test(line.toUpperCase());
+    if (line.includes('-->') || (/^NOTE/).test(line)) {
+      const isNote = (/^NOTE/).test(line);
       t.times = isNote ? "" : line;
       t.tag = isNote ? TRANSCRIPT_CUE_TYPES.note : TRANSCRIPT_CUE_TYPES.timedCue;
-      t.line = isNote ? line : '';
+      // Make sure there is a single space separating NOTE from the comment for single or multi-line comments
+      t.line = isNote ? line.replace(/^NOTE\s*/,'NOTE ') : '';
       i++;
```

Comment:
Does this still need to be in the loop with adding `i++` to the for loop declaration?

Reply:
Yes, it advances to the next line so the while loop test starts with the line after the start of the cue/note.